packages/features/ee/teams/components/TeamsListing.tsx

```diff
@@ -59,66 +58,26 @@ export function TeamsListing() {
   return (
     <>
       {!!errorMessage && <Alert severity="error" title={errorMessage} />}
-      {invites.length > 0 && (
-        <div className="mb-4">
-          <h1 className="mb-2 text-lg font-medium">{t("open_invitations")}</h1>
-          <TeamList teams={invites} />
-        </div>
-      )}
-      {isLoading && <SkeletonLoaderTeamList />}
-      {!teams.length && !isLoading && (
-        <>
-          {!isCalcom ? (
-            <div className="-mt-6 rtl:ml-4 md:rtl:ml-0">
-              <div
-                className="flex w-full justify-between overflow-hidden rounded-lg pt-4 pb-10 md:min-h-[295px] md:pt-10"
-                style={{
-                  background: "url(/team-banner-background.jpg)",
-                  backgroundSize: "cover",
-                  backgroundRepeat: "no-repeat",
-                }}>
-                <div className="mt-3 px-8 sm:px-14">
-                  <h1 className="font-cal text-3xl">{t("calcom_is_better_with_team")}</h1>
-                  <p className="my-4 max-w-sm text-gray-600">{t("add_your_team_members")}</p>
-                  <div className="space-y-2 rtl:space-x-reverse sm:space-x-2">
-                    <ButtonGroup>
-                      <Button color="primary" href={`${WEBAPP_URL}/settings/teams/new`}>
-                        {t("create_team")}
-                      </Button>
-                      <Button color="secondary" href="https://go.cal.com/teams-video" target="_blank">
-                        {t("learn_more")}
-                      </Button>
-                    </ButtonGroup>
-                  </div>
-                </div>
-              </div>
-              <div className="mt-4 grid-cols-3 md:grid md:gap-4">
-                {features.map((feature) => (
-                  <div
-                    key={feature.title}
-                    className="mb-4 min-h-[180px] w-full rounded-md bg-gray-50 p-8 md:mb-0">
-                    {feature.icon}
-                    <h2 className="font-cal mt-4 text-lg">{feature.title}</h2>
-                    <p className="text-gray-700">{feature.description}</p>
-                  </div>
-                ))}
-              </div>
-            </div>
-          ) : (
-            <EmptyScreen
-              Icon={Icon.FiUsers}
-              headline={t("no_teams")}
-              description={t("no_teams_description")}
-              buttonRaw={
-                <Button color="secondary" href={`${WEBAPP_URL}/settings/teams/new`}>
-                  {t("create_team")}
-                </Button>
-              }
-            />
-          )}
-        </>
-      )}
-      {teams.length > 0 && <TeamList teams={teams} />}
+      <UpgradeTip
+        title="calcom_is_better_with_team"
```

Comment:
@sean-brydon @PeerRich it seems this line (and `description` below) have resulted in the translation no longer being displayed. I've created [this issue](https://github.com/calcom/cal.com/issues/6838) as a way to track it.